### ISSUE TITLE
mysql optional, maybe with --no-mysql

### ISSUE THREAD

**Issue by george-vieira** [GreenCom-Networks/centreon-install#6]
I've built a couple of servers with this setup (thanks for the large effort too), but one annoying thing is how much I have to edit the script. The main one atm is the database, I have a remote database for performance and large disk size reasons but the script forces me to set it up.

Other thing with the script, I tend to forget sometimes the mysql  MY_PASSWORD isnt changed.
I think it might be an idea to check the value if it isn't changed and exit with error, forcing the user to change it.

**Comment by albundy83** [comment]
Yes that's a good idea also, for moment I'm stuck with the new version that Centreon released.
They don't work when I try to update an existing installation.
